packages/web/src/pages/chat-page/components/CreateChatUserResult.tsx

```diff
@@ -162,16 +160,11 @@ export const CreateChatUserResult = (props: UserResultComposeProps) => {
   useEffect(() => {
     if (
       currentUserId &&
-      supportingMap[currentUserId]?.[user.user_id] &&
-      !supportersMap[user.user_id]
+      supporters.some((s) => s.sender.user_id === currentUserId)
     ) {
-      dispatch(fetchSupportersForUser({ userId: user.user_id }))
+      // No need to fetch supporters if we're already a supporter
     }
-  }, [dispatch, currentUserId, supportingMap, supportersMap, user])
-
-  useEffect(() => {
-    dispatch(fetchPermissions({ userIds: [user.user_id] }))
-  }, [dispatch, user])
+  }, [currentUserId, supporters])
```

Comment:
is this a no-op?

Reply:
good find ^ i think this got extra updated due to ai